src/api/ans.ts

```diff
@@ -30,7 +30,7 @@ export class ANS {
    *
    * @returns MoveAddressType if the name is owned, undefined otherwise
    */
-  async getOwnerAddress(args: { domainName: string; subdomainName?: string }): Promise<MoveAddressType | undefined> {
+  async getOwnerAddress(args: { domainName: string; subdomainName?: string }): Promise<AccountAddress | undefined> {
     return getOwnerAddress({ aptosConfig: this.config, ...args });
```

Comment:
Hmm, just thinking about this from a user perspective.  Would this want to just be `name` and parse the `domain` and `subdomain` out of it?  Might be less confusing / more adaptable.

For example, if I just throw my ANS name test.greg.apt into a website, I hope it'll be able to just figure it out, without the website builder having to always write that parsing logic themselves.